app/scripts/services/routes.js

```diff
@@ -72,6 +73,10 @@ angular.module("openshiftConsole")
           }
           warnings.push(message);
         }
+
+        if (wildcardPolicy === 'Subdomain' && ingress.wildcardPolicy !== wildcardPolicy) {
```

Comment:
Add a comment here that this will only show up for older routers that don't reject the route based on wildcard policy.